data/ui/preferences.blp

```diff
@@ -10,39 +10,6 @@ template $PreferencesWindow : Adw.PreferencesWindow {
     icon-name: "applications-system-symbolic";
     title: _("General");
 
```

Comment:
The top page of the preference window does look slightly empty now. But I have not a super great suggestion to make that much better. I guess we can keep that as-is for now.

Reply:
My thought was to have this page be the second and move the action preference. But thats more suited for another pr imo.

Reply:
Yeah I was thinking this menu now needs a bit of a redesign anyway, but agreed that this should be done in a seperate PR.